osquery/tables/networking/linux/interface_ip.cpp

```diff
@@ -0,0 +1,82 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#include <boost/algorithm/string/trim.hpp>
+#include <boost/assign/list_of.hpp>
+#include <boost/lexical_cast.hpp>
```

Comment:
Why do you need those two includes? 
```c++
#include <boost/assign/list_of.hpp>
#include <boost/lexical_cast.hpp>
```

Reply:
It's just leftover from my first draft :/ 
I'll remove them.